python/ray/tests/test_logging.py

```diff
@@ -526,7 +526,6 @@ def print_after(_obj):
 
 def test_log_redirect_to_stderr(shutdown_only):
     log_components = {
-        ray_constants.PROCESS_TYPE_DASHBOARD: "Dashboard head grpc address",
```

Comment:
We should keep this but check a different log message

Reply:
Use "Starting dashboard metrics server on port" instead.